lib/tasks/evm.rake

```diff
@@ -55,6 +55,19 @@ namespace :evm do
     EvmApplication.update_stop
   end
 
+  desc "Determine the deployment scenario"
+  task :deployment_status => :environment do
+    status_to_code = {
+      "new_deployment" => 0,
+      "new_replica"    => 1,
+      "redeployment"   => 2,
+      "upgrade"        => 3
```

Comment:
yeah, I honestly have no idea what those numbers mean 😕 

Clearly, they're exit codes but from where?  Can you put a link or a comment explaining them...